libraries/Microsoft.Bot.Schema/TokenResponse.cs

```diff
@@ -29,12 +26,9 @@ public TokenResponse()
         /// <summary>
         /// Initializes a new instance of the TokenResponse class.
         /// </summary>
-        /// <param name="connectionName">The connection name</param>
-        /// <param name="token">The user token</param>
-        /// <param name="expiration">Expiration for the token, in ISO 8601
-        /// format (e.g. "2007-04-05T14:30Z")</param>
-        public TokenResponse(string connectionName = default(string), string token = default(string), string expiration = default(string))
+        public TokenResponse(string channelId = default(string), string connectionName = default(string), string token = default(string), string expiration = default(string))
```

Comment:
Breaking change. Can't take this change.

Reply:
@Jeffders We've been (manually) moving all new parameters to the end of the functions/constructors, if you move channelId to the end of this we can take this change.